src/main.cpp

```diff
@@ -391,6 +391,7 @@ void setup()
 
     // We need to scan here to decide if we have a screen for nodeDB.init() and because power has been applied to
     // accessories
+    delay(500);  // Introduce a delay before scanning for I2C devices
```

Comment:
Alternate suggestion: repeat the bus scan/test for keyboards in the cannedMessages init. This 500msec will go on top the 5s delay we have waiting for the serial port.

Reply:
I added a way to rescan right after and it works. Removed the delay

Reply:
nvm, it gave me lots of errors. reverting to delay